packages/sanity/src/_internal/cli/server/getStudioEnvironmentVariables.ts

```diff
@@ -4,29 +4,46 @@ import {loadEnv} from '@sanity/cli'
 const envPrefix = 'SANITY_STUDIO_'
 
 /**
- * Get environment variables prefixed with SANITY_STUDIO_, as an object.
- *
- * Specify `options.prefix` to add a prefix to the environment variable keys,
- * eg: `getStudioEnvironmentVariables({prefix: 'process.env.'})`
- *
- * Specify `options.envFile` to include environment variables from dotenv files (`.env`),
- * in the same way the studio does. A `mode` must be specified, usually `development`
- * or `production`, which will load the corresponding `.env.development` or `.env.production`.
- * To specify where to look for the dotenv files, specify `options.envFile.envDir`.
+ * The params for the `getStudioEnvironmentVariables` function that gets Studio focused environment variables.
  *
- * Specify `options.jsonEncode` to JSON-encode the values, which is handy if you want to pass
- * this to a bundlers hardcoded defines, such as Vite's `define` or Webpack's `DefinePlugin`.
+ * @public
+ */
+export interface studioEnvVariablesOptions {
```

Comment:
Interfaces should be PascalCase
```suggestion
export interface StudioEnvVariablesOptions {
```